src/webgl/p5.Shader.js

```diff
@@ -329,7 +329,7 @@ p5.Shader = class {
         this.setUniform('uPerspective', 1);
```

Comment:
I was testing out the example and realized that we still force perspective when using a default camera. Maybe we can take out this if statement and just use the else branch?